packages/js/src/ui/components/elements/Preferences/Preferences.tsx

```diff
@@ -22,6 +22,18 @@ export const Preferences = () => {
     return { globalPreference, workflowPreferences };
   });
 
+  createEffect(() => {
+    // Register the names as localizable
+    setDynamicLocalization((prev) => ({
+      ...prev,
+      ...allPreferences().workflowPreferences?.reduce<Record<string, string>>((acc, preference) => {
+        acc[preference.workflow!.id] = preference.workflow!.name;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** as we agreed, please update this to the `identifier`